src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@yegor256 pls, be more specific: it's possible to have many ctors and use one of them as a primary ctor. So secondary ctors call the primary with some code: `Elegant Objects` book val. 1 `1.2 Make one ctor primary`, page 29. Here is Cash example with just some code in secondary ctor.